dev-server/documents/html/sidebar-external-container.mustache

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Annotatable iframe test</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+      .container {
+        display: flex;
+      }
+      #my-sidebar {
+        flex: 0 0 20%;
+      }
+      iframe {
+        flex: 1;
+        height: 400px;
+        resize: both;
+        overflow: auto;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Sidebar on external container test</h1>
+    <p>
+      The sidebar is placed inside the element with `#my-sidebar` id. The iframe
```

Comment:
```suggestion
      The sidebar is placed inside the element with `my-sidebar` id. The iframe
```

Pedantic comment - the `#` is not part of the ID. In a CSS selector it indicates an ID selector.